deb-x64/Dockerfile

```diff
@@ -1,3 +1,7 @@
+FROM ubuntu as CURL_GETTER
```

Comment:
We probably want this on ARM too, right?

Reply:
We don't have the problem there, though, since everything is a bit newer... I don't have a strong opinion either way.

Reply:
👍, let's keep it like this. We can clean it up on a future PR if we want to have the same version everywhere

Reply:
Ended up changing it :P